build-tools/packages/build-cli/src/commands/release/setPackageTypesField.ts

```diff
@@ -122,8 +120,10 @@ export default class SetReleaseTagPublishingCommand extends PackageCommand<
 		await super.run();
 
 		if (this.packageList.packagesUpdated.length === 0) {
-			this.errorLog(`No updates in package.json for ${this.flags.types} release tag`);
-			this.exit();
+			this.errorLog(
+				`##vso[task.logissue type=error]No updates in package.json for ${this.flags.types} release tag`,
+			);
```

Comment:
I vote against hardcoding ADO syntax here, would rather keep throwing the plain message and having the necessary error-detection code in the pipeline steps that call `flub release`.